### DISCUSSION TITLE
Project specific key bindings?

### DISCUSSION THREAD

**Discussion by malkomalko** [zed-industries/zed#9090] [Feature Requests]
Just getting into Zed and loving it. It doesn't look like project specific key bindings are supported. I was thinking it might be nice to add a .zed/keymap.json in your project the same way you can add a .zed/settings.json for project specific settings.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
